index.ts

```diff
@@ -194,37 +194,35 @@ function suiteClassCallback(target: SuiteCtor, context: TestFunctions) {
             const shouldPending = method[pendingSymbol];
             const parameters = method[parametersSymbol] as TestParams[];
 
-            /* istanbul ignore else */
-            if (testName) {
-                if (parameters) {
-                    const nameForParameters = method[nameForParametersSymbol];
-                    parameters.forEach((parameterOptions, i) => {
-                        const { mark, name, params } = parameterOptions;
-
-                        let parametersTestName = `${testName}_${i}`;
-                        if (name) {
-                            parametersTestName = name;
-                        } else if (nameForParameters) {
-                            parametersTestName = nameForParameters(params);
-                        }
-
-                        const shouldSkipParam = shouldSkip || (mark === Mark.skip);
-                        const shouldOnlyParam = shouldOnly || (mark === Mark.only);
-                        const shouldPendingParam = shouldPending || (mark === Mark.pending);
-
-                        const testFunc = ((shouldPendingParam || shouldSkipParam) && context.it.skip)
-                                || (shouldOnlyParam && context.it.only)
-                                || context.it;
-
-                        applyTestFunc(testFunc, parametersTestName, method, [params]);
-                    });
-                } else {
-                    const testFunc = ((shouldPending || shouldSkip) && context.it.skip)
-                        || (shouldOnly && context.it.only)
-                        || context.it;
-
-                    applyTestFunc(testFunc, testName, method, []);
-                }
+            // assert testName is truthy
```

Comment:
just remove comment as it is misleading

Reply:
rationale here is that ``if (parameters)`` is documentation in itself.

Reply:
There was an `if (testName) {` with istanbul ignored else.
I think to replace these `// assert testName` with some sort of `assert(testName, "Expected test name to be set")` in next commits.

Reply:
ah, ok, so it is a TODO that is under cover :smile: